### PR TITLE
[bento][amp-iframe] changes to publish to npm

### PR DESCRIPTION
**Pull request by dmanek**
Partial for #35147

This PR includes the following changes:
1. Rename `amp-iframe` to `bento-iframe`
2. addition of `amp-iframe.css`
3. Publish to npm

Still pending: README.md

### CODE REVIEW

**Comment by dmanek** [comment]
@CrystalOnScript some minor updates to `amp-iframe.md`, appreciate if you can PTAL.

File: extensions/amp-iframe/amp-iframe.md [hunk 1]
```diff
@@ -243,7 +241,8 @@ window.addEventListener('message', function (event) {
 
 The intersection message would be sent by the parent to the iframe in the format of IntersectionObserver entry wheneve there is intersectionRatio change across thresholds [0, 0.05, 0.1, ... 0.9, 0.95, 1].
 
-## Iframe & Consent Data
+<!-- TODO: uncomment when Bento amp-consent (v1.0) is ready -->
```
Comment (caroqliu) [inline extensions/amp-iframe/amp-iframe.md:244]: Let's create an issue or something to refer to here?

File: extensions/amp-iframe/amp-iframe.md [hunk 2]
```diff
@@ -92,6 +86,10 @@ The reasons for this policy are that:
 -   `amp-iframe` has no fully iframe controlled resize mechanism.
 -   Viewability information may not be available to `amp-iframe`.
 
+### Standalone use outside valid AMP documents
+
+Bento AMP allows you to use AMP components in non-AMP pages without needing to commit to fully valid AMP. You can take these components and place them in implementations with frameworks and CMSs that don't support AMP. Read more in our guide [Use AMP components in non-AMP pages](https://amp.dev/documentation/guides-and-tutorials/start/bento_guide/). To find the standalone version of `amp-iframe`, see [`bento-iframe`](./1.0/README.md).
```
Comment (caroqliu) [inline extensions/amp-iframe/amp-iframe.md:91]: Until README.md exists, this will have a dead link issue in CI

**Review by caroqliu — APPROVED** [review]

File: extensions/amp-iframe/amp-iframe.md (hunk 1 above)
Comment (dmanek) [reply extensions/amp-iframe/amp-iframe.md:244]: Done. Created #36211 & linked in todo.

File: extensions/amp-iframe/amp-iframe.md (hunk 2 above)
Comment (dmanek) [reply extensions/amp-iframe/amp-iframe.md:91]: That's right. README.me is added in #36210 and will be merged soon.

**Review by CrystalOnScript — APPROVED** [review]

**Pull request merged**